traces-observer-service/.env.example

```
@@ -1,24 +1,28 @@
-OPENSEARCH_USERNAME=admin
-OPENSEARCH_PASSWORD=admin
+LOG_LEVEL=INFO
 
-# Query Configuration
-DEFAULT_SPAN_QUERY_LIMIT=1000
-LOG_LEVEL=DEBUG
+# -----------------------------------------------------------------------------
+# Observer Service Configuration
+# -----------------------------------------------------------------------------
+OBSERVER_BASE_URL=http://localhost:8085
+IDP_TOKEN_URL=http://localhost:8090/oauth2/token
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IDP_TOKEN_URL key should go before the OBSERVER_BASE_URL key

(UnorderedKey)